added searchbar to Dashboard.vue

joakgus opened the pull request:
updaterade lines 4->21 in dashboard.vue för att få en searchbar  och avgränsare

willeasp reviewed (changes requested):
The search bar is a little low
Put the searchbar aligned to the left
Maybe add an explaining text like "Filter by name"

joakgus commented:
changed the searchbar from dashboard.vue to dashboardnavbar.vue , made small change to scss to show searchbar when on mobile, changed icon for about us and the text when navigating to aboutus to about us.

willeasp reviewed (approved):
Verry nice!!!
